test/module/shared_model/validators/validators_fixture.hpp

```diff
@@ -146,6 +146,10 @@ class ValidatorsTest : public ::testing::Test {
     // Iterate through all concrete types
     boost::for_each(range, [&](auto field) {
```

Comment:
let's avoid null pointers please!
```suggestion
     boost::for_each(
        range | boost::adaptors::filtered([this](const auto &field) {
          return not ignored_fields_.count(field.full_name());
        }),
        [&](auto field) {
```

Reply:
fixed